manifests/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -19,111 +19,155 @@ spec:
   - name: v1
     schema:
       openAPIV3Schema:
-        description: "Network describes the cluster's desired network configuration. It is consumed by the cluster-network-operator. \n Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer)."
+        description: "Network describes the cluster's desired network configuration.
+          It is consumed by the cluster-network-operator. \n Compatibility level 1:
+          Stable within a major release for a minimum of 12 months or 3 minor releases
+          (whichever is longer)."
```

Comment:
This file seems to have gotten line-wrapped but none of the actual API changed? What caused that? (More importantly, "is it going to accidentally get flipped back the next time someone changes something"?)